src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -498,4 +502,18 @@ private function renderDisableMultistorePage(int $productId = null): Response
             ),
         ]);
     }
+
+    /**
+     * @return bool
+     */
+    protected function shouldRedirectToV1(): bool
```

Comment:
I really don't think this should be `protected`. We know for sure that this method will be deleted when we'll remove the feature flag and make this product v2 page the default one, so why should we allow to override it?

Reply:
If the dev wants to define it as default, we should allow it. Let the power to the developer to decide.

Reply:
no I think @atomiix is right, those methods are temporary, we already know that we will remove them sooner than later, it's probably not a good idea to make them overridable

Reply:
Temporary but available during 1 year (at less). We are a platform for developers. I don't decide for them.

Reply:
well they can access one product page or the other by making the switch in the feature flag page, there's no need to override the method

if they override it, we put them in a BC break situation once we remove it

Reply:
if they really need to, they can also override the `indexAction` method, and cancel the call to the redirection, and/or do something else instead